docs/getting_started/what_do_i_need.md

```diff
@@ -5,16 +5,16 @@ In order to run Zigbee2MQTT we need the following hardware:
 
 | Name | Price | Picture |
 | ------------- | ------------- | ------------- |
-| Supported Zigbee adapter | 5$ - 40$ | ![CC2531 USB sniffer](../images/cc2531.jpg) |
-| [Supported Zigbee device(s)](../information/supported_devices.md) | - | ![Zigbee devices](../images/xiaomi_sensors.jpg) |
-| PC or Raspberry Pi (*) to run bridge | - | ![Raspberry Pi](../images/pi.jpg) |
+| A supported [Zigbee adapter](../information/supported_adapters.md) (with Zigbee coordinator firmware) | 5$ - 40$ | ![CC2531 USB sniffer](../images/cc2531.jpg) |
```

Comment:
"(with Zigbee coordinator firmware)" can be left out, it's not a requirement as below is explained how to do this.

Reply:
OK